persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_11/generated/entities/generated_client.bal

```diff
@@ -4,24 +4,93 @@
 // It should not be modified by hand.
 
 import ballerina/persist;
+import ballerina/sql;
+import ballerinax/mysql;
 
 public client class EntitiesClient {
-    *persist:AbstractPersistClient;
+
+    private final mysql:Client dbClient;
+
+    private final map<persist:SQLClient> persistClients;
+
+    private final map<persist:Metadata> metadata = {medicalneed: {entityName: "MedicalNeed", tableName: 'MedicalNeed ', needId: {columnName: "needId", 'type: int}, itemId: {columnName: "itemId", 'type: int}, beneficiaryId: {columnName: "beneficiaryId", 'type: int}, period: {columnName: "period", 'type: time:Civil}, urgency: {columnName: "urgency", 'type: string}, quantity: {columnName: "quantity", 'type: int} keyFields: ["needId"]}};
```

Comment:
Shall we format this code more readable manner

Reply:
updated in [aa95e9c](https://github.com/ballerina-platform/persist-tools/pull/106/commits/aa95e9c0a62f306236f4dc03ae1d6f60593a1b6e)